api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/util/SchemaVersionChecker.java

```diff
@@ -43,9 +45,17 @@ public void waitForSchemaUpdates(Session session, String keyspace, String versio
                             "WHERE config_id = ? AND name = ?");
                 }
                 ResultSet resultSet = session.execute(getVersion.bind(configId, configName));
-                if (!resultSet.isExhausted() && version.equals(resultSet.one().getString(0))) {
-                    logger.infof("Hawkular Metrics is up to date at version %s", version);
-                    return;
+                if (!resultSet.isExhausted()) {
+                    String schemaVersion = resultSet.one().getString(0);
+                    if(version.equals(schemaVersion)) {
+                        logger.infof("Hawkular Metrics is up to date at version %s", version);
```

Comment:
I would change this slightly to, 

"Hawkular Metrics schema is up to date at version %s"